### PR TITLE
[Assembly] `<VideoFeature />`

### PR DESCRIPTION
**Pull request by EnMod**
🎟️ [Asana Task](https://app.asana.com/0/1201874072258661/1202983318984725/f)
🔍 [Preview Link](https://react-components-git-asm-videofeature-hashicorp.vercel.app/components/video-feature)

---

## Description

This is an assembly collecting changes for a new component: `<VideoFeature />` 

### PR Checklist 🚀

Items in this checklist may not may not apply to your PR, but please consider each item carefully.

- [x] Add Asana and Preview links above.
- [x] Conduct thorough self-review.
- [x] Add or update tests as appropriate.
- [x] Conduct reasonable cross browser testing for both compatibility and responsive behavior (We have a [Sauce Labs](https://app.saucelabs.com/) account for this, if you don't have access, just ask!).
- [ ] Conduct reasonable accessibility review (use the [WAS](https://accessible.org/Web-Accessibility-Standards-WAS-2.pdf) as a guide or an [axe browser plugin](https://www.deque.com/axe/) until we establish more formal checks).
- [x] Identify (in the description above) and document (add Asana tasks on [this board](https://app.asana.com/0/1100423001970639/list)) any technical debt that you're aware of, but are not addressing as part of this PR.

### CODE REVIEW

**Comment by EnMod** [comment]
Reopening

File: package-lock.json [hunk 1]
```diff
@@ -1904,8 +1904,7 @@
     },
```
Comment (alexcarpenter) [inline package-lock.json:1904]: @EnMod are these changes to top-level package-lock expected?

Comment (EnMod) [reply package-lock.json:1904]: @alexcarpenter Ah, these snuck through.  Probably worth reverting the changes removing `"resolved"` and `"integrity"` keys

Comment (EnMod) [reply package-lock.json]: Ok changes have been reverted as of these latest commits:
https://github.com/hashicorp/react-components/pull/783/commits/17f413d5f9f682a430fc8e6c3fdf5240e3f22f79
https://github.com/hashicorp/react-components/pull/783/commits/988a47b6cb834e3b300621578455c3ca885136af

**Review by alexcarpenter — APPROVED** [review]

**Comment by EnMod** [comment]
Ah good eye! Just removed bottom padding for the description here:
https://github.com/hashicorp/react-components/pull/783/commits/84ec21940003b33a89fd9c1728d4c49977e78128

**Pull request merged**